sdk/keyvault/keyvault-keys/src/keysModels.ts

```diff
@@ -2,32 +2,117 @@
 // Licensed under the MIT License.
 
 import * as coreHttp from "@azure/core-http";
-import {
-  JsonWebKey,
-  JsonWebKeyOperation,
-  JsonWebKeyCurveName,
-  JsonWebKeyType
-} from "./core/models";
+import { JsonWebKeyOperation, JsonWebKeyCurveName, JsonWebKeyType } from "./core/models";
 import { DeletionRecoveryLevel } from "./core/models";
 
 /**
  * @interface
  * An interface representing the key client. For internal use.
  */
 export interface KeyClientInterface {
+  /**
+   * Recovers the deleted key in the specified vault. This operation can only be performed on a
+   * soft-delete enabled vault.
+   */
   recoverDeletedKey(name: string, options?: GetDeletedKeyOptions): Promise<KeyVaultKey>;
+  /**
+   * The get method gets a specified key and is applicable to any key stored in Azure Key Vault.
+   * This operation requires the keys/get permission.
+   */
   getKey(name: string, options?: GetKeyOptions): Promise<KeyVaultKey>;
+  /**
+   * The delete operation applies to any key stored in Azure Key Vault. Individual versions
+   * of a key can not be deleted, only all versions of a given key at once.
+   */
   deleteKey(name: string, options?: coreHttp.OperationOptions): Promise<DeletedKey>;
+  /**
+   * The getDeletedKey method returns the specified deleted key along with its properties.
```

Comment:
Nit: inconsistent use of generalized name (like "get" on line 19 above) where here you actually use the method name.